Update Metadata + Screenshots of Activities

Gabbar8055 opened the pull request.

godiard commented:
Why change the license in the metadata? @quozl this has been decided by the sugarlabs community?

godiard commented:
@vipulgupta2048, I see you requested the license change, why?

quozl commented:
Thanks for the pull request.  Reviewed.

* [ ] do not change the license metadata without explanation; we're waiting for your explanation, but my quick look at the source code shows `helpactivity.py`, `viewtoolbar.py` and `setup.py` are GPLv2+, and unless you have a good reason to change along with the authority as copyright owner to do so, please don't, (as a contractor for One Laptop per Child, I'm able to act for the copyright owner, so I'm interested in what you think the licence should change to),
* [ ] please use the same size for all screenshots, so that they look the same when presented together; the existing screenshot is 1200x900, which was the native resolution of the OLPC XO laptop, but 1024x768 is a reasonable alternative,
* [ ] each screenshot has a cost down the line; so please don't add more screenshots than are needed; for instance the third screenshot doesn't seem remarkable.

2443faa8d3f4f5d4e8bc0ea8f00f3cf63454f8d0 was where Seth Woodward changed license metadata from GPLv2 to GPLv3 as part of a merge.  This may have been a mistake, I'm not sure.

Gabbar8055 commented:
I have changed the license because vipulgupta2048,  requested to change the license. 
I have added the screenshot with 1024x768 size.
I have removed the third screenshot.

quozl commented:
* [ ] text is blurred, as if the screenshot was resized instead of being captured at the resolution.

quozl commented:
Thanks.  Reviewed.  Still has blur, as if using a different resolution for display.  Also has blue bar on top.
